pkg/db/models.go

```diff
@@ -43,6 +43,9 @@ type Campaign struct {
 	// a campaign should make requests with this interval in between them
 	ScheduleInterval time.Duration `json:"schedule_interval"`
 
+	// current status of the campaign, used to pause/cancel/resume without deletion
+	Status string `json:"status"`
+
```

Comment:
Can we either make this an [enum](https://www.ribice.ba/golang-enums/) or pull the constants from `pkg/db/client.go` into this file?